td/python-scripting/pandas-df/scripts/examples.py

```diff
@@ -0,0 +1,40 @@
+# Because we do not allow custom image currently, this is how you can add 3rd party
+# libraries instead
+import sys, os
+sys.path.append('/home/td-user/.local/lib/python3.6/site-packages')
+os.system(f"{sys.executable} -m pip install --user pandas-td")
+
+import pandas_td as td
+import tdclient
+
+con = td.connect(apikey=os.environ['TD_API_KEY'], endpoint=os.environ['TD_API_SERVER'])
+
+def read_td_table(database_name, table_name, engine_name = 'presto', limit=1000):
+    engine = td.create_engine("{}:{}".format(engine_name, database_name), con=con)
```

Comment:
[nits] how about using f-string?

```suggestion
    engine = td.create_engine(f"{engine_name}:{database_name}", con=con)
```